app/controllers/application_controller.rb

```diff
@@ -1,5 +1,6 @@
 class ApplicationController < ActionController::Base
   include GDS::SSO::ControllerMethods
+  before_filter :require_signin_permission!
```

Comment:
Modern rails api prefers `before_action` instead.  I don't know if `before_filter` is deprecated yet, but it can't hurt to use the new api.